chemprop/nn/metrics.py

```diff
@@ -45,15 +45,12 @@
 
 class Metric(LossFunction):
     """
-    __init__(self, task_weights: Tensor):
+    __init__(self, task_weights: ArrayLike = 1.0):
```

Comment:
why is the method signature inside the docstring?

Reply:
I thought I had to specify which method the Parameters shown were for, which was difficult given the method is defined in LossFunction. I now see that a class docstring can have Parameters specified, like in `MPNN`. will remove this now.